### PR TITLE
feat: add Tolgee template

### PR DESCRIPTION
**Pull request by CodeWithKamal**
This PR adds a new template for Tolgee. The template has been tested and works with Traefik for routing. Also, added appropriate environment configurations for Docker.

### CODE REVIEW

**Comment by CodeWithKamal** [comment]
### Steps to use the template:

1. Deploy the template
2. Wait for few minutes for Tolgee to boot up.
3. Goto the `TOLGEE_HOST` value
4. Login with username/email: `admin` and password: `admin`

There you go!

File: apps/dokploy/templates/tolgee/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  app:
+    image: tolgee/tolgee:v3.80.4
+    volumes:
+      - ./data:/data
+      - ./config.yaml:/config.yaml
+    ports:
+      - '8080:8080'
```
Comment (Siumauricio) [inline apps/dokploy/templates/tolgee/docker-compose.yml]: Can you leave as `8080`

**Review by Siumauricio — APPROVED** [review]

**Review by Siumauricio — CHANGES_REQUESTED** [review]
The Templates works fine, just review the comments

Thanks!

File: apps/dokploy/templates/tolgee/docker-compose.yml (hunk 1 above)
Comment (CodeWithKamal) [reply apps/dokploy/templates/tolgee/docker-compose.yml]: I'm new to docker and learning so, not sure what to do.

do you mean like this?

```
ports:
    - '8080'
```

Comment (Siumauricio) [reply apps/dokploy/templates/tolgee/docker-compose.yml]: Yes!

**Comment by CodeWithKamal** [comment]
done!

**Review by Siumauricio — APPROVED** [review]
Thanks!

**Comment by Siumauricio** [comment]
run `pnpm check` to format all the files

**Comment by CodeWithKamal** [comment]
what should I do now ?

**Comment by Siumauricio** [comment]
Fixed, just ran again `pnpm check`

**Comment by CodeWithKamal** [comment]
done, fixed the lint issue.

**Review by Siumauricio — APPROVED** [review]

**Pull request merged**